src/administrator/libraries/action.php

```diff
@@ -188,7 +188,7 @@ public function bind(&$array)
 	 *
 	 * @since __DEPLOY_VERSION__
 	 */
-	public static function loadActionByCode(String $code, String $client)
+	public static function loadActionByCode($code, $client)
```

Comment:
Why did you remove the typecasting ? Do you see any integer only values of code or client ?